.controlplane/entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+# Runs before the main command
+
+# No action is required for this project to be done before running the app.
```

Comment:
This entrypoint file doesn't have any use here. I kept it to keep the structure.